Doc/using/configure.rst

```diff
@@ -1097,7 +1097,8 @@ CPython project) this is usually the ``all`` target. The
 all`` will build. The three choices are:
 
 * ``profile-opt`` (configured with ``--enable-optimizations``)
-* ``build_wasm`` (configured with ``--with-emscripten-target``)
+* ``build_wasm`` (chosen if the host platform is either ``wasm32-wasi`` or
```

Comment:
@brettcannon Am I correct in saying `wasm-wasip1` is now the preferred host identifier?

Reply:
If we want to be more specific, it is set to `build_wasm` if `ac_sys_system` is either `Emscripten` or `WASI` so if `host` matches either `*-*-wasi*` or `*-*-emscripten`.

Reply:
Ok - so then I'd suggest:
```suggestion
* ``build_wasm`` (chosen if the host platform matches ``wasm32-wasi*`` or
```